src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -31,8 +31,8 @@ public void Configure(IAppHostBuilder appBuilder)
 						{"Logging:LogLevel:Default", "Warning"}
 					});
 				})
-				.UseMauiServiceProviderFactory(true)
-				//.UseServiceProviderFactory(new DIExtensionsServiceProviderFactory())
+				//.UseMauiServiceProviderFactory(true)
+				.UseServiceProviderFactory(new DIExtensionsServiceProviderFactory())
```

Comment:
We will need to see if there is a fast way to handle multiple service registration and if it will conflict with the single registration we want for things like handlers.